Gemfile

```diff
@@ -22,6 +22,8 @@ gem 'elasticsearch-transport', '~> 6.0'
 gem 'flickraw', '~> 0.9.8'
 gem 'hashie', '~> 3.5.7'
 gem 'mock_redis', '~> 0.17.3'
+# Keeping a version of nokogiri that supports both Rails 6 and 7
+gem 'nokogiri', '~> 1.15.3'
```

Comment:
We don't need to lock nokigiri's version

Reply:
@stevenbarragan  - Ah yes, forgot about this one. I forgot to remove this as I no longer needed this.

Reply:
@stevenbarragan  - I remembered why I did this. In circleci, we have a matrix for the ruby version of 2.7.5 where we have a nokogiri depedency error during bundle install step: 

```
The following gems are missing
 * nokogiri (1.16.0)
 * mini_portile2 (2.8.5)
Install missing gems with `bundle install`
Fetching gem metadata from https://rubygems.org/.........
nokogiri-1.16.0 requires ruby version >= 3.0.0, which is incompatible with the
current version, 2.7.5
```

I am thinking we either state the explicit older version of nokogiri, `1.15.3` or we can remove the ruby 2.7.5 version from the matrix in circleci, thoughts?

Reply:
Let's get rid of the ruby 2.7.5 and the 3.0.5 version checks instead. We don't need them anymore.